compiler/rustc_builtin_macros/src/format_foreign.rs

```diff
@@ -579,14 +579,7 @@ pub mod printf {
         }
     }
 
-    fn is_digit(c: char) -> bool {
-        match c {
-            '0'..='9' => true,
-            _ => false,
-        }
-    }
-
-    fn is_flag(c: char) -> bool {
+    fn is_flag(c: &char) -> bool {
         match c {
             '0' | '-' | '+' | ' ' | '#' | '\'' => true,
```

Comment:
This could be turned into `matches!` while you're at it :)

Reply:
Different branch I'm still working on ;)